components/Cards/index.js

```diff
@@ -17,3 +17,48 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+    .then(response => {
+        // console.log(response.data)
+        response.data.articles.javascript.forEach(article => createCard(article));
```

Comment:
Can you think of a way to programmatically loop over the response data to make all the cards?